ns/log.n3

```diff
@@ -0,0 +1,396 @@
+#   n3 definition of some Semantic Web terms
+#
+# These raise the level above RDF's pure relational data,
+# and Web Ontology's ontological level, to allow rules to be expressed
+# and inference done. They connect the inference to the web,
+# allowing data to be fetched from resources elsewhere, and remote
+# servers to be queried.
+# 
+# $Id: log.n3,v 1.25 2010-03-26 20:35:14 timbl Exp $
+
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix doc: <http://www.w3.org/2000/10/swap/pim/doc#>.
+@prefix log: <http://www.w3.org/2000/10/swap/log#>.
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+@prefix : <#>.
+
+log:N3Document a rdfs:Class;
+    rdfs:comment    """A document which, which parsed as Notation3
+as defined in general by http://www.w3.org/DesignIssues/Notation3.html
+and this schema, conveys the intent of the author by the semantics defined
+on those specifications, and the semantics defined by the specifications
+of any other identifiers used in the document.
+""" .
+
+################################################## Syntactic level
+#
+#  It isn't clear to me that these should be described as Properties
+#  and Classes.  They operate at a level of symbol identity, not identity of the things referred to.
+#
+
+log:equalTo a rdf:Property;
+    rdfs:comment
+"""True if the subject and object are the same RDF node (symbol or literal).
+Do not confuse with owl:sameAs.
+A cwm built-in logical operator, RDF graph level.
+""".
+
+log:notEqualTo a rdf:Property; owl:inverse log:equalTo;
+    rdfs:comment
+"""Equality in this sense is actually the same URI.
+A cwm built-in logical operator.
+""".
+
+log:rawType a rdf:Property;
+    rdfs:range log:Type;
+    rdfs:comment """
+This is a low-level language type, one of log:Formula, log:Literal, 
+log:List, log:Set or log:Other. 
+Example: log:semanticsOrError returns either a formula or a string,
+and you can check which using log:rawType.
+""".
+
+log:Type a rdfs:Class
+    rdfs:comment """
+The class of Notation3-specifiec resource types.
+""";
+    vs:term_status "unstable".
+
+log:Formula a rdfs:Class; rdfs:subClassOf log:Type.
+log:List    a rdfs:Class; rdfs:subClassOf log:Type, rdf:List.
+log:String  a rdfs:Class; rdfs:subClassOf log:Type.      # Replace with xsd:string or rdfs:Literal?
+log:Other   a rdfs:Class; rdfs:subClassOf log:Type.     # Not one of the above.
+
+#   The following are inherent in the translation of N3 syntax to quad representation:
+
+# FIXME: no such thing as rdf:SyntaxArc.
+log:forSome a rdf:SyntaxArc;  #@@@@@   NOT a real property. A pseudo property
```

Comment:
Note, there is no `rdf:SyntaxArc` class.

Both of these were in place, and may not bear a good relationship to what is suggested in our builtin wishlist.

Reply:
unsure what you mean by this:
>may not bear a good relationship to what is suggested in our builtin wishlist.

Reply:
These were in the original namespace document, but commented out. I'm not sure if they are the same as suggested in the wish-list document.